src/openforms/js/components/admin/form_design/registrations/camunda/FormFields.js

```diff
@@ -153,105 +263,50 @@ const FormFields = ({processDefinitions, formData, onChange}) => {
-            allowBlank
-          />
-        </CustomFieldTemplate>
-        <CustomFieldTemplate
-          id="camundaOptions.processDefinitionVersion"
-          label={intl.formatMessage({
-            defaultMessage: 'Version',
-            description: "Camunda 'process definition version' label",
+      <div style={{display: 'flex', gap: '10px', alignItems: 'start'}}>
+        <ActionButton
+          text={intl.formatMessage({
+            description: 'Link label to open registration options modal',
+            defaultMessage: 'Configure options',
           })}
-          rawErrors={null}
-          errors={null} // TODO
-          rawDescription={intl.formatMessage({
-            description: "Camunda 'process definition version' help text",
-            defaultMessage:
-              'Which version of the process definition to start. The latest version is used if not specified.',
-          })}
-          required={false}
-          displayLabel
-        >
-          <Select
-            name="processDefinitionVersion"
-            choices={versionChoices}
-            value={processDefinitionVersion ? `${processDefinitionVersion}` : ''}
-            onChange={onFieldChange}
-            allowBlank
-          />
-        </CustomFieldTemplate>
+          type="button"
+          onClick={() => setBaseModalOpen(!baseModalOpen)}
+        />
 
-        <CustomFieldTemplate
-          id="camundaOptions.manageProcessVars"
-          displayLabel={false}
-          rawErrors={null}
-        >
-          <ActionButton
-            text={intl.formatMessage({
-              description: 'Open manage camunda process vars modal button',
-              defaultMessage: 'Manage process variables',
-            })}
-            type="button"
+        <span style={{display: 'flex', flexDirection: 'column', gap: '5px'}}>
+          <ManageSimpleVarsButton
+            numVars={processVariables.length}
             onClick={() => setSimpleVarsModalOpen(!simpleVarsModalOpen)}
           />
-          &nbsp;
-          <FormattedMessage
-            description="Managed Camunda process vars state feedback"
-            defaultMessage="{varCount, plural,
-                            =0 {}
-                            one {(1 variable mapped)}
-                            other {({varCount} variables mapped)}
-                        }"
-            values={{varCount: processVariables.length}}
-          />
-        </CustomFieldTemplate>
+        </span>
 
-        <CustomFieldTemplate
-          id="camundaOptions.manageComplexProcessVars"
-          displayLabel={false}
-          rawErrors={null}
-        >
-          <ActionButton
-            text={intl.formatMessage({
-              description: 'Open manage complex camunda process vars modal button',
-              defaultMessage: 'Complex process variables',
-            })}
-            type="button"
+        <span style={{display: 'flex', flexDirection: 'column', gap: '5px'}}>
```

Comment:
Same as the `ManageSimpleVarsButton` span, why not add the `span` to the `ManageComplexVarsButton` component?

Reply:
moving it. in my mind it didn't matter because it's the same code being copy pasted in similar places, there's still the same amount of inline styles :shrug: